src/app/lib/utilities/getMostReadUrls/index.js

```diff
@@ -0,0 +1,9 @@
+export const getMostReadEndpoint = ({ service, variant }) =>
```

Comment:
This file looks great, just wondering if we could have just one function and depending on the environment `process.env.SIMORGH_APP_ENV` then return the right path.

Reply:
the getLocalMostReadEndpoint is for when we are on storybook and it would be difficult to differentiate storybook environment from our others.

Reply:
Fair enough 👍